aptos-move/dynamic-transaction-composer/src/builder.rs

```diff
@@ -0,0 +1,545 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Code to generate compiled Move script from a series of individual Move calls.
+//!
+//! Defined a struct called `TransactionComposer`. With `TransactionComposer`, you will be able to add new Move calls
+//! to it via the `add_batched_call` api. The code will perform a light weight type check to make sure the ability and
+//! type safety of the Move calls will be held. After adding the calls, you will be able to consume the `TransactionComposer`
+//! and get the corresponding `CompiledScript` via the `generate_batched_calls` api.
+
+#[cfg(test)]
+use crate::BatchedFunctionCall;
```

Comment:
As I said above, we decided to not use "batch". Please get the naming consistent.